docs/powergate/devnet.md

```diff
@@ -0,0 +1,169 @@
+
+## Devnet with Powergate
+
+If you're interested in running Powergate to experiment with the CLI or APIs, the fastest way is to replace the Lotus client dependency with a running devnet. This will run the Powergate on a Lotus client connected to an embedded network of miners.
+
+### Installation
+
+Clone the Powergate and `cd` into the project
+
+```bash
+git clone git@github.com:textileio/powergate.git
+cd powergate
+```
+
+### Setup
+
+A default setup is available in a `docker-compose` configuration shipped with the Powergate. With the default setup, you will run Powergate connected to a local devnet with 512Mib sectors and instantly available gRPC API or CLI that don't require any extra config flags 🎊
+
+**Run the docker-compose**
+
+Docker files for the Powergate are all contained in the folder, `/docker`.
+
+```bash
+cd docker
+make devnet
+```
+
+If this is your first time running the Powergate, Docker will download the required instances before any Powergate setup begins. Downloads are dependent on your bandwidth and may take **a few minutes**, but wont be required for subsequent runs of the Powergate.
+
+Once running, you will begin to see log outputs, including those of the embedded miner.
+
+```bash
+lotus_1      | 2020-05-29T20:35:08.644Z	WARN	miner	miner/miner.go:177\
+mined block in the past	{"block-time": "2009-01-01T04:44:30.000Z",\
+"time": "2020-05-29T20:35:08.644Z", "duration": 359999438.64444387}
+```
+
+When complete, you will have a fully functional Powergate (`powd`), a Lotus devnet, and an IPFS node wired correctly together to start using.
+
+### Create a deal and store a file
+
+Now that your Powergate is running on devnet, all the CLI and API commands are the same as using it in production mode, just your deals will store faster (and disappear when you delete the devnet).
+
+**Install the CLI**
+
+From the root of the `powergate` repo, you can build the CLI from the latest code. This will install the Powergate CLI, `pow`, on your machine.
+
+```bash
+make build
+```
+
+Test your installation.
+
+```bash
+pow --help
+```
+
+**Create an FFS instance**
+
+FFS is the most common API for interacting with the Powergate. To use the API, you must first create an empty FFS instance, which will,
+
+- Create a new wallet address on Lotus.
+- Track and manage deals associated with that address in the Powergate FFS.
+- Create an API token for using that FFS (and address) over the Powergate API.
+
+```Bash
+pow ffs create
+```
+
+???+ success
+
+    ```Bash
+    Instance created with id 0ac0fb4d-581c-4276-bd90-a9aa30dd4cb4 and token 883f57b1-4e66-47f8-b291-7cf8b10f6370
+    ```
+
+**Generate a CID for your file on IPFS**
+
+To store data on the cold layer (Filecoin), you first need to make it available on IPFS. You can do that quickly by using the Powergate to temporarily add it to the embedded IPFS node.
+
+```bash
+pow ffs addToHot -t 883f57b1-4e66-47f8-b291-7cf8b10f6370 myfile
```

Comment:
Maybe would be nice to rename `addToHot` to `import`? The camelcase and length feel wierd.
Opinions?